src/tui/light_windows.go

```diff
@@ -0,0 +1,131 @@
+//+build windows
+
+package tui
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+
+	"github.com/junegunn/fzf/src/util"
+	"golang.org/x/crypto/ssh/terminal"
+	"golang.org/x/sys/windows"
+)
+
+var (
+	oldStateInput  uint32
+	oldStateOutput uint32
+)
+
+// IsLightRendererSupported checks to see if the Light renderer is supported
+func IsLightRendererSupported() bool {
+	var oldState uint32
+	// enable vt100 emulation (https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences)
+	if err := windows.GetConsoleMode(windows.Stderr, &oldState); err != nil {
+		return false
+	}
+	canSetVt100 := windows.SetConsoleMode(windows.Stderr, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING) == nil
+	windows.SetConsoleMode(windows.Stderr, oldState)
+	return canSetVt100
+}
+
+func (r *LightRenderer) initPlatform() error {
+	IsLightRendererSupported()
```

Comment:
Return value of this function isn't checked. Intentional?

Reply:
This was addressed in a later commit.